main/reference/ertp-api/purse.md

```diff
@@ -37,7 +38,22 @@ quatloosPurse2 = quatloosIssuer.makeEmptyPurse();
 ```
 
 ## `purse.getCurrentAmount()`
-- Returns: `{Amount}`
+
+Gets the `purse`'s current balance.
```

Comment:
`purse` rather than **purse**?

Reply:
I'm not sure what revision you're suggesting; the text had `purse` rather than **purse**. However, I noticed that I revised this API entry w/out revising any other entries. So I reverted this entry back to the previous format until I have a chance to update all the API entries to whatever new format we decide upon.